### PR TITLE
New cask: Ableton Live Intro 9.7.4

### PR DESCRIPTION
**Pull request by ghost**
Adapted from `ableton-live-standard.rb`

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
Please complete all of the steps in the PR template.

**Comment by decorate** [comment]
Remade `cask` locally and tested. All good.

File: Casks/ableton-live-intro.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  version '9.7.4'
+  sha256 '7062febe6ac590b48a936de876f08fb7e91b38a14016c5dd885a710991eb21f0'
+
+  url "http://cdn-downloads.ableton.com/channels/#{version}/ableton_live_intro_#{version}_64.dmg"
+  name 'Ableton Live Intro'
+  homepage 'https://www.ableton.com/en/live/'
+
+  app "Ableton Live #{version.major} Intro.app"
+
+  zap delete: '~/Library/*/*[Aa]bleton*',
```
Comment (reitermarkus) [inline Casks/ableton-live-intro.rb]: Won't this remove files for other Ableton software?

Comment (decorate) [reply Casks/ableton-live-intro.rb]: It does though only another version of Ableton. I lifted the zap stanza from the existing Ableton casks (Lite, Standard, Suite).

**Pull request merged**